### DISCUSSION TITLE
How to change index.html meta title?

### DISCUSSION THREAD

**Discussion by nalakawula** [mmistakes/minimal-mistakes#2483] [Q&A] [answered]
I clone your master branch then add my post and my pages.
The meta title for home page (index.html) is weird, because it use title from about.md.
Kindly check here https://zealous-sammet-315091.netlify.com/, it show 
```html
<!-- begin _includes/seo.html --><title>Posts by Collection - sumarsono.com</title>
```
**Posts by Collection** is title from about page.

My question is, how to add correct title to my home page without add `<h1 id="page-title" class="page__title" itemprop="headline">` in my home page.

Thank you.

**Comment by mmistakes** [comment, marked as answer]
You add the page title using YAML Front Matter. The `.html` or `.md` file you're using for the Posts by Collection page has a `title` key that you can modify.

For example, the demo "Posts by Collection" page has the [following front matter](https://github.com/mmistakes/minimal-mistakes/blob/master/docs/_pages/collection-archive.html#L3).

```
title: "Posts by Collection"
```

Change the the text to what you want.
